app/uk/gov/hmrc/cdsreimbursementclaimfrontend/connectors/RejectedGoodsMultipleClaimConnector.scala

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cdsreimbursementclaimfrontend.connectors
+
+import akka.actor.ActorSystem
+import cats.syntax.eq._
+import com.google.inject.ImplementedBy
+import com.google.inject.Inject
+import play.api.libs.json.Format
+import play.api.libs.json.Json
+import uk.gov.hmrc.cdsreimbursementclaimfrontend.journeys.RejectedGoodsMultipleJourney
+import uk.gov.hmrc.cdsreimbursementclaimfrontend.utils.HttpResponseOps._
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.http.HttpClient
+import uk.gov.hmrc.http.HttpReads.Implicits._
+import uk.gov.hmrc.http.HttpResponse
+import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
+
+import javax.inject.Singleton
+import scala.concurrent.ExecutionContext
+import scala.concurrent.Future
+import scala.concurrent.duration._
+import play.api.Configuration
+
+@ImplementedBy(classOf[RejectedGoodsMultipleClaimConnectorImpl])
+trait RejectedGoodsMultipleClaimConnector {
```

Comment:
This is unnecessary cruft which goes against the teams coding standards. See https://github.com/alexandru/scala-best-practices/blob/master/sections/2-language-rules.md#24-should-not-define-useless-traits

Reply:
I presume it is here only because we are testing using Mockito, and mocking an interface is standard practice, is it?

Reply:
Removing the trait would work fine with Mockito, we are using ScalaMock. This is another reason to dislike ScalaMock.